modules/calypsoify/class.jetpack-calypsoify.php

```diff
@@ -253,14 +253,23 @@ public function add_custom_menus() {
 	public function enqueue() {
 		wp_enqueue_style( 'calypsoify_wpadminmods_css', plugin_dir_url( __FILE__ ) . 'style.min.css', false, JETPACK__VERSION );
 		wp_style_add_data( 'calypsoify_wpadminmods_css', 'rtl', 'replace' );
-        wp_style_add_data( 'calypsoify_wpadminmods_css', 'suffix', '.min' );
-
+		wp_style_add_data( 'calypsoify_wpadminmods_css', 'suffix', '.min' );
+
+		$icons = array(
+			'checkmark'   => get_gridicon( 'gridicons-checkmark' ),
```

Comment:
I am not sure you'll be able to rely on `get_gridicon` here; this function is available on Atomic sites because WooCommerce Calypso Bridge is present there, but it does not exist on other, non-Atomic Jetpack sites.

Reply:
@jeherve I added `gridicons` via composer. Should any changes be made to the dev setup instructions? I had to run composer update to get `gridicons` to install.

Reply:
That should be fine. I have one concern though: since we're adding the full gridicons repo to Jetpack, we'll end up shipping a lot of unnecessary files within Jetpack:

![image](https://user-images.githubusercontent.com/426388/62034242-ef336880-b1ed-11e9-9cdf-b07ebcd949b6.png)

Would there be a way to include a stripped down version of the repo, only including the code that we need? 
Somewhat related, I wonder if we could take that opportunity to publish that stripped down version of the Gridicons package to packagist, alongside the other a8c packages:
https://packagist.org/packages/automattic/
I realize this may be easier said than done, as the repo was not organized as your typical package from the start: there is no `src` directory we could include while excluding everything else.

If we can't reorganize the repo so we can easily get rid of the unnecessary files, could you edit `.svnignore` in the Jetpack repo to exclude the files we won't need in Jetpack?

Reply:
@jeherve I'm running `NODE_ENV=production yarn build-client` and the `vendor/automattic/gridicons` folder isn't being created/populated. Is there something I need to add to have the module added?